webapp/components/layout/WidgetFactory.tsx

```diff
@@ -113,6 +114,9 @@ export default class WidgetFactory {
       case "analysis": {
         return <NetPyNEPlots model={"simConfig.analysis"}/>
       }
+      case "experiments": {
+        return <Experiments />
```

Comment:
spacing of "/>" is different compared to previous line, noticed that we still need eslint support for typescript #233